substrate/validator-sets/pallet/src/lib.rs

```diff
@@ -377,6 +381,8 @@ pub mod pallet {
     BadSignature,
     /// Validator wasn't registered or active.
     NonExistentValidator,
+    /// Deallocation would take the stake below what is required.
+    NotEnoughStake,
```

Comment:
"DeallocationWouldRemoveEconomicSecurity"? Keep it line with the similar ones above?